central/complianceoperator/v2/scanconfigurations/datastore/datastore_impl_test.go

```diff
@@ -434,7 +434,7 @@ func (s *complianceScanConfigDataStoreTestSuite) TestScanConfigurationProfileExi
 			configID:    uuid.NewV4().String(),
 			profiles:    []string{"ocp4-cis-1-4-0"},
 			clusters:    []string{s.clusterID1},
-			isErrorTest: true,
+			isErrorTest: false,
```

Comment:
This only way to invoke this now would be to create a scan config with `ocp4-cis-1-4-0` and `ocp4-cis-1-4-0`, right?

That's probably not possible through the UI with the checkboxed implementation, but it may still be something we want to validate using the API tests.

Reply:
it could be sometimes there already exist a `ocp4-cis-1-4-0` config, we will block that if you create a new one contains `ocp4-cis-1-4-0`

Reply:
And just to double check - we have this case covered in our e2e tests with the following?

https://github.com/stackrox/stackrox/blob/c60ffaf765e91a6e4d82671c314fe25ec9ca3e53/tests/compliance_operator_v2_test.go#L214-L235

Reply:
correct, just added in another e2e to test for versioned profile as well